### PR TITLE
#RI-2063 - dynamically display command groups

### PR DESCRIPTION
**Pull request by romansergeenkosofteq**
#RI-2064 - rename ZSET to Sorted Set

### CODE REVIEW

File: redisinsight/ui/src/components/cli/components/cli-search/CliSearchFilter/CliSearchFilter.tsx [hunk 1]
```diff
@@ -35,17 +38,22 @@ const CliSearchFilter = ({ submitFilter, isLoading }: Props) => {
     setTypeSelected('')
   }, [matchedCommand])
 
-  const options: EuiSuperSelectOption<string>[] = FILTER_GROUP_TYPE_OPTIONS.map(
+  const groupOptions = [...commandGroups].sort().map((group: string) => ({
+    text: (GROUP_TYPES_DISPLAY as any)[group] || group.replace(/_/, ' '),
```
Comment (zalenskiSofteq) [inline redisinsight/ui/src/components/cli/components/cli-search/CliSearchFilter/CliSearchFilter.tsx:42]: I think better use `.replaceAll()`, or add flag `/g` to regexp

File: redisinsight/ui/src/components/group-badge/GroupBadge.tsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ const GroupBadge = ({ type, name = '', className = '' }: Props) => (
     data-testid={`badge-${type} ${name}`}
   >
     <EuiText style={{ color: 'var(--euiTextSubduedColorHover)' }} className="text-uppercase" size="xs">
-      {GROUP_TYPES_DISPLAY[type] ?? type}
+      {(GROUP_TYPES_DISPLAY as any)[type] ?? type.replace(/_/, ' ')}
```
Comment (zalenskiSofteq) [inline redisinsight/ui/src/components/group-badge/GroupBadge.tsx:18]: the same

File: redisinsight/ui/src/components/cli/components/cli-search/CliSearchFilter/CliSearchFilter.tsx (hunk 1 above)
Comment (romansergeenkosofteq) [reply redisinsight/ui/src/components/cli/components/cli-search/CliSearchFilter/CliSearchFilter.tsx]: yes, thanks! Done

File: redisinsight/ui/src/components/group-badge/GroupBadge.tsx (hunk 2 above)
Comment (romansergeenkosofteq) [reply redisinsight/ui/src/components/group-badge/GroupBadge.tsx]: done

**Review by zalenskiSofteq — APPROVED** [review]

**Pull request merged**